### PR TITLE
ADDED: Sports Registration Form Initial Version

### PR DESCRIPTION
**Pull request by sivabalan18102001**
Sports Registration Form Initial Version Submitted

### CODE REVIEW

**Comment by NavinAT** [comment]
Code Merged to Main

File: Sivabalan/15-02-2023 Task/Next page.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      <script>  
+          let text = localStorage.getItem("testJSON");
+          let obj = JSON.parse(text);
+          for(let i=0;i<obj.length;i++){
+            var user = obj[i];
+            j=i+1;
+            document.write('<tr><td>'+j+'</td><td>'+ user.fname+'</td><td>'+user.mname+'</td><td>'+user.lname+'</td><td>'+user.Court_Name+'</td><td>'+user.Registration_date+'</td><td>'
+                            +user.Gender+'</td><td>'+user.Mobile_number+'</td><td>'+user.Address+'</td><td>'+user.Email+'</td></tr>');
+          }
+        function editUser(){
```
Comment (NavinAT) [inline Sivabalan/15-02-2023 Task/Next page.html:40]: Move this Script code to separate JavaScript and indent the code properly.

File: Sivabalan/15-02-2023 Task/Style for Next second page.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+body{
```
Comment (NavinAT) [inline Sivabalan/15-02-2023 Task/Style for Next second page.css:1]: Indent the CSS properly

File: Sivabalan/15-02-2023 Task/Style sheet for 15-02-2023 Task.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
+*{
```
Comment (NavinAT) [inline Sivabalan/15-02-2023 Task/Style sheet for 15-02-2023 Task.css:1]: Indent the CSS properly

File: Sivabalan/15-02-2023 Task/Task 15-02-2023.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            <th><label>Email:</label></th>
+            <th><input type="email" id="email" name="email" required></th>
+        </tr>
+    </table>
+    <button id="submit" type="submit" onclick="submitForm()">Register</button><br><br>
+    <button id="reset" type="button" onclick="resetlocalStorage()">Reset Local Storage</button>
+    </form>
+   </div>
+    <script>
+        function submitForm() {
```
Comment (NavinAT) [inline Sivabalan/15-02-2023 Task/Task 15-02-2023.html:77]: Move this Script to separate JS file.

File: Sivabalan/15-02-2023 Task/User Edit Page.html [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+        <tr>
+            <th><label>Email:</label></th>
+            <th><input type="email" id="email" name="email" required></th>
+        </tr>
+    </table>
+    <button id="submit" type="submit" onclick="updateDetails()">Save</button><br><br>
+    <button id="cancel" type="button" onclick="cancelEdit()">Cancel</button>
+    </form>
+    <script>
+            var obj = localStorage.getItem("testJSON");
```
Comment (NavinAT) [inline Sivabalan/15-02-2023 Task/User Edit Page.html:76]: Move this Script to separate file.

**Review by NavinAT — COMMENTED** [review]
Done the Initial review please do the changes.

File: Sivabalan/15-02-2023 Task/Style for Next second page.css (hunk 2 above)
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Style for Next second page.css:1]: completed

File: Sivabalan/15-02-2023 Task/Style sheet for 15-02-2023 Task.css (hunk 3 above)
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Style sheet for 15-02-2023 Task.css:1]: completed

**Pull request merged**